### ISSUE TITLE
Make playerctl dependency optional

### ISSUE THREAD

**Issue by Grumph** [BlingCorp/bling#48]
I use some modules from bling, but not playerctl.

At startup, I have the following error:
```
stack traceback:
        [C]: in function 'assert'
        /usr/share/lua/5.3/lgi/namespace.lua:158: in function 'lgi.namespace.require'
        (...tail calls...)
        ...umph/.config/awesome/external/bling/signal/playerctl.lua:22: in main chunk
        [C]: in function 'require'
        /home/grumph/.config/awesome/external/bling/signal/init.lua:1: in main chunk
        [C]: in function 'require'
        /home/grumph/.config/awesome/external/bling/init.lua:8: in main chunk
        [C]: in function 'require'
        /home/grumph/.config/awesome/rc.lua:54: in main chunk
error: /usr/share/lua/5.3/lgi/namespace.lua:158: Typelib file for namespace 'Playerctl' (any version) not found
```
Of course one way to avoid this could be to install `playerctl` on my system, but I don't want to because I don't need it.

There should be a way to avoid loading this playerctl module, with a warning somewhere to inform the user to install the `playerctl` package when explicitly using the playerctl module.

**Comment by Nooo37** [comment]
Bran just reported the same problem. Do you also use nix by chance? 

That surely has to be fixed anyway. The only fix I can think of right now would be to only call the `require` statement when the enable function fo the playerctl signal gets called. #46 is connected but won't fix this issue by itself so we'll keep both issues.

**Comment by JavaCafe01** [comment]
@Grumph This is addressed in #49, do you want to try and test that PR out?

**Comment by Grumph** [comment]
Good for me with PR #49 :+1:
